shared-data/python/opentrons_shared_data/labware/labware_definition.py

```diff
@@ -252,6 +294,16 @@ class Group(BaseModel):
     )
 
 
+class InnerLabwareGeometry(BaseModel):
+    frusta: List[BoundedSection] = Field(
+        ...,
+        description="A list of all of the sections of the well that have a contiguous shape",
+    )
+    bottom_shape: Optional[Hemisphere] = Field(
```

Comment:
```suggestion
    bottomShape: Optional[Hemisphere] = Field(
```

`camelCase` for JSON models, plz

(and of course update the `TypedDict` version the same way)